source/connect.txt

```diff
@@ -204,6 +204,23 @@ For ``tls`` connections:
 
         mongosh "mongodb://mongodb0.example.com:28015" --tls
 
+Connect to a Specific Database
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To connect to a specific database, specify a database in your
+`connection string URI path
+<https://docs.mongodb.com/manual/reference/connection-string/>`__. If
+you do not specify a database in your URI path, you connect to a Test
```

Comment:
[nit] Since this is case-sensitive, I think it's important to say a `test` database. Also, since "test database" can mean different things in different contexts, I think we should say: "you connect to a database named `test`".

Use double backticks (\`\`) around `test` to get the monospaced styling so it stands out.